docs/source/periodic_table.rst

```diff
@@ -0,0 +1,80 @@
+Periodic Table
+==============
+
+Full access to the periodic table is also available within QCPortal. This
+includes mass, atomic number, symbol, and name information. Data is indexed
+many ways so that many possible aliases can lead to a single quantity. For
+example, to obtain the mass (in a.m.u) of of Carbon atomic number, symbol,
+isotope symbol, and name can all be passed:
+
+.. code-block:: python
+
+    >>> qcel.periodictable.to_mass(6)
+    12.0
+    >>> qcel.periodictable.to_mass("C")
+    12.0
+    >>> qcel.periodictable.to_mass("C12")
+    12.0
+    >>> qcel.periodictable.to_mass("Carbon")
+    12.0
+
+A brevity of infomation can be accessed in this manner. Taking 'Carbon' as an example for mass information:
```

Comment:
brevity --> variety?

Reply:
I was thinking bevy, but variety is better.

Reply:
either :-)